src/main/java/io/github/thebusybiscuit/slimefun4/utils/SlimefunUtils.java

```diff
@@ -120,7 +123,8 @@ public static boolean isSoulbound(@Nullable ItemStack item, @Nullable World worl
         if (item != null && item.getType() != Material.AIR) {
             ItemMeta meta = item.hasItemMeta() ? item.getItemMeta() : null;
 
-            if (hasSoulboundFlag(meta)) {
+            SlimefunItem rune = SlimefunItems.SOULBOUND_RUNE.getItem();
+            if (hasSoulboundFlag(meta) && rune != null && !rune.isDisabled() && (world == null || !rune.isDisabledIn(world))) {
```

Comment:
This should probably be nested inside the if-statement. If the flag is not present then there is no need to perform any additional checks.

Or a much better approach would be to put the `hasSoulboundFlag` at the end. Since it might be more expensive computation-wise. Depends on what the cheaper operation is. The null check and `.isDisabled()` are probably the cheapest ones. So let's try it that way I guess. This should allow the code to disregard unneeded operations

```suggestion
            SlimefunItem rune = SlimefunItems.SOULBOUND_RUNE.getItem();
            if (rune != null && !rune.isDisabled() && (world == null || !rune.isDisabledIn(world)) && hasSoulboundFlag(meta)) {
```